ci/ci.sh

```diff
@@ -154,7 +154,7 @@ prepare_docker() {
     EXPOSE 8000
     EXPOSE 10001
     RUN pip install /${wheel}[serve]
-    RUN sudo apt update && sudo apt install curl -y
```

Comment:
should we revert this change?

Reply:
I think when I run it locally, it failed (`apt update`).
The reason it failed is because some upstream is not responsive.
This is just to make it  more robust. As long as curl can be installed, we shall be good.